core/sr-std/without_std.rs

```diff
@@ -16,6 +16,26 @@
 
 #[cfg(feature = "nightly")]
 #[doc(hidden)]
+
+pub use core::borrow;
```

Comment:
Why did you move this?

Reply:
a posteriori the move is unnecessary, will undo that. Just wanted to group the alloc stuff